atlasdb-config/src/test/java/com/palantir/atlasdb/config/AtlasDbConfigSerializationTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.junit.Test;
+
+import com.palantir.atlasdb.memory.InMemoryAtlasDbConfig;
+import com.palantir.remoting2.config.ssl.SslConfiguration;
+
+public class AtlasDbConfigSerializationTest {
```

Comment:
Ah, right - there's only deserialization (I think when I wrote this I was thinking 'test the serialized form', but we don't generate on-disk configs as you point out), so it's probably clearer as `...DeserializationTest`. 

Will do this for the timelock server config one too, which is similar.